.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[build]
```

Comment:
Moved to repo root so that flags are appleid when you do `cargo build` from repo root or from VSCode.